raiden/tests/scenarios/mfee2_proportional_fees.yaml

```diff
@@ -68,10 +68,12 @@ scenario:
 
             # Fees are set to 10 TKN per 1000 transferred. Hence for 10000, 100 TKN fee
             # should be paid, plus a 1% fee of fees per mediator.
-            # balance_minus_fee = deposit - transfer_amount - (transfer_amount / prop_fee * amount_of_hubs_to_target)
-            # balance_minus_fee_and_fee_margin = balance_minus_fee - (transfer_amount / prop_fee / 100 * fee_margin)
-            - assert: {from: 0, to: 1, total_deposit: 100_000, balance: 89_789, state: "opened"}
-            - assert: {from: 1, to: 0, total_deposit: 0, balance: 10_211, state: "opened"}
+            # Fee for second mediator = 10000 * 1% = 100
+            # Fee for first mediator = 10100 * 1% = 101
+            # Fee margin = ceil(201 * 3% + 10000 * 0.05%) = 12
```

Comment:
ceil() rundet auf, oder? https://de.wikipedia.org/wiki/Abrundungsfunktion_und_Aufrundungsfunktion

Coole Beschreibung! Danke, dass Du Dich da durch kämpfst

Reply:
Yes, that helps keeping the margin high enough when the amounts get small.